app/javascript/components/donations/Form.tsx

```diff
@@ -12,7 +12,7 @@ import { useQueryCache } from 'react-query'
 
 const TabsContext = createContext<TabContext>({
   current: 'subscription',
-  switchToTab: () => {},
+  switchToTab: () => null,
```

Comment:
What does this change do?

Reply:
Nothing really, it only pleases the TypeScript linter. See: https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/no-empty-function.md